config/webpack.target.services.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const path = require('path')
+const { DefinePlugin } = require('webpack')
+const nodeExternals = require('webpack-node-externals')
+
+const TARGET_DIR = path.resolve(__dirname, '../targets/')
+
+module.exports = function(production, app) {
+  var entry = {
+    onPhotoUpload: path.resolve(TARGET_DIR, './photos/services/onPhotoUpload')
+  }
+  var plugins = [
+    new DefinePlugin({
+      __TARGET__: JSON.stringify('services')
```

Comment:
Will we really need this? We use the `__TARGET__` var to differentiate between web and mobile, but for a service script I'm not sure it will be needed :)

Reply:
Right, I removed it